Makefile

```diff
@@ -346,9 +346,10 @@ $(PKG): release-only
 pkg: $(PKG)
 
 pkg-upload: pkg
-	ssh $(STAGINGSERVER) "mkdir -p staging/$(DISTTYPEDIR)/$(FULLVERSION)"
-	scp -p node-$(FULLVERSION).pkg $(STAGINGSERVER):staging/$(DISTTYPEDIR)/$(FULLVERSION)/node-$(FULLVERSION).pkg
-	ssh $(STAGINGSERVER) "touch staging/$(DISTTYPEDIR)/$(FULLVERSION)/node-$(FULLVERSION).pkg.done"
+	ssh $(STAGINGSERVER) "mkdir -p nodejs/$(DISTTYPEDIR)/$(FULLVERSION)"
+	chmod 664 node-$(FULLVERSION).pkg
```

Comment:
yes because everything is uploaded to the server to user `staging` but it's user `dist` that has the permission to promote them, i.e. you can't promote a build if you only have `staging`, you have to have `dist` like all of the releasers do, without umask `002` the cleanup process doesn't work properly, we've been putting up with this for io.js releases for a long time now and this is my chance to fix the thing.

sadly I couldn't come up with a clean way to apply a umask to `scp` connections for an individual user on the server-side, `rsync` might offer some better options there but it's not on the Windows machines so ...